images/win/scripts/Tests/Miniconda.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "Miniconda" {
+    It "Miniconda Environment variables is set. " {
+        Get-EnvironmentVariable "CONDA" | Should -Not -BeNullOrEmpty
+    }
+    
+    It "Miniconda $env:CONDA\<PathTest> " -TestCases @(
+        @{ PathTest = "python.exe" }
+        @{ PathTest = "Scripts\conda.exe" }
+    ) {
+        Join-Path ${env:CONDA} $PathTest | Should -Exist
```

Comment:
Do we want to do something like `conda.exe --version` and `python.exe --version`?

Reply:
thank you, added.